components/alert/alert.js

```diff
@@ -53,9 +53,6 @@ class Alert extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 			:host([type="error"]) .d2l-alert-highlight {
 				background-color: var(--d2l-color-feedback-error);
 			}
-			:host([type="warning"]) .d2l-alert-highlight {
```

Comment:
I don't think we can safely remove this. We can mark it as deprecated in the docs but there seem to be quite a few places currently using it.

Reply:
Right now this would remove the colored band entirely, so I suspect something has to happen either way, but it's a question for @geurts  I think. I could assign `[type="warning"]` to the same style as the default if that's the goal.

Reply:
Yeah if there's a clear migration path for those alerts we should take it... I'm guessing to the default. There's also the [MVC version](https://search.d2l.dev/xref/lms/lp/framework/web/D2L.LP.Web.UI/Desktop/Controls/Alert/AlertTypes.cs?r=7c3878c6#7) of this control that'll need updating as well.